src/index.html

```diff
@@ -17,16 +17,30 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      action="#"
+      class="box__search1"
+      data-qa="big"
+    >
+      <input
+        type="text"
+        data-qa="keypress"
+        placeholder="Try “Los Angeles“"
+        class="big__search"
+      />
+    </form>
 
```

Comment:
Add an empty line between the two form elements to separate multiline sibling blocks of HTML and improve readability.